surefire-report-parser/pom.xml

```diff
@@ -18,69 +18,70 @@
   ~ under the License.
   -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>org.apache.maven.surefire</groupId>
-    <artifactId>surefire</artifactId>
-    <version>3.0.0-M6-SNAPSHOT</version>
-  </parent>
+    <parent>
+        <groupId>org.apache.maven.surefire</groupId>
+        <artifactId>surefire</artifactId>
+        <version>3.0.0-M6-SNAPSHOT</version>
+    </parent>
```

Comment:
is indentation correct ... looks like change from 2 -> 4 space?

Reply:
4 is correct
I am using Maven profile in IDEA.

Reply:
@slawekjaranowski 
But maven-shared-utils has CLI class, platform specific impl. We need it. Other classes are minor. If we could extract only that as sources and not bytecode, and if we substituted `DirectoryScanner ` with NIO Files, then yes, it would be possible to get right of shaded classes.

Reply:
There is not much methods and classes we use, we can write our own, few util methods. The problem is with CLI.

Reply:
https://maven.apache.org/developers/conventions/code.html
https://maven.apache.org/developers/maven-idea-codestyle.xml

4 - for java
2 - for xml

Reply:
I thought about `surefire-shared-utils` not maven-shared-utils ...

I now there are place where we must shade package from maven-shared-utils and other but I hope it can be done in the last module in the tree where maven-shared-utils is used ... but it is another story

Reply:
@slawekjaranowski 
We shade the `maven-shared-utils` into `surefire-shared-utils`, and much more.
I can imaging the dependency `maven-shared-utils` with type `sources` extracting CLI to generated sources, and adding new `DirectoryScanner` based on NIO in `src/main/java`. This can be done but you need to have a spare time to write unit tests for the scaner and several trivial utility methods.